test/any/datastore/rate_limited_DAO-test.es6.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('RateLimitedDAO', function() {
+  let rateLimitedDAO;
+  foam.CLASS({
```

Comment:
Please bump this up into the `beforeEach` and ensure that `node_modules/foam2/test/helpers/testcontext.js` is loaded by Jasmine. Each test should run in its own context, so the test class should be registered each time.